compiler/luci/service/src/Nodes/CircleReshape.test.cpp

```diff
@@ -39,7 +39,7 @@ TEST(CloneNodeTest, clone_Reshape)
   ASSERT_EQ(node_reshape->newShape()->dim(1), cloned_reshape->newShape()->dim(1));
 }
 
-TEST(ShapeRuleTest, reshape_by_input_const_static)
+TEST(ShapeRuleTest, reshape_by_const_static)
```

Comment:
Just for curiosity, Is there a reason for changing this test name?

Reply:
Just for future naming consistency :)
I thought it could be too verbose if I keep naming like the old one.